app/assets/stylesheets/govuk_publishing_components/components/_subscription-links.scss

```
@@ -77,3 +79,44 @@
     transform-origin: bottom left;
   }
 }
+
+.gem-c-subscription-links--with-feed-box {
+  .gem-c-subscription-links__list-item {
+    vertical-align: middle;
+  }
+
+  .gem-c-subscription-links__link {
+    &:not(.gem-c-subscription-links__link--feed) {
+      padding: govuk-spacing(2) govuk-spacing(2) govuk-spacing(2) 0;
+      border: 1px solid transparent;
+      border-bottom: 1px solid govuk-colour("dark-grey");
+
+      &:focus {
+        border-bottom: 0;
```

I think that changing the bottom border to 0 when focused is causing a very slight layout shift in Safari and Chrome. A quick experiment in the devtools, could changing the border colour to transparent work?
```suggestion
        border-bottom-color: transparent;
```